constants/training-course-purpose.js

```diff
@@ -1,6 +1,6 @@
 const trainingCoursePurpose = {
-    'higher-education': 'Higher Education',
-    'training': 'Training',
+    higherEducation: 'Higher education',
```

Comment:
in many places I saw higher-education called, Could you please have a look the changes to higherEducation not going to cause any trouble in other places 

aspel-workspace/packages/asl-projects/client/helpers/index.js
aspel-workspace/packages/asl-schema/test/migrations/20200710110948_training_and_education_ppls_false.js
aspel-workspace/packages/asl-schema/migrations/20200710110948_training_and_education_ppls_false.js

Reply:
new const which is created here is used in all the places that this ticket needs. 

I had a look at the files above, from what I see, new changes shouldn't impact old changes. Let's see duing testing

Reply:
@Mo - please let me know if you have any more comments, I am happy to raise another PR if needed. I am going merge it now because I need this change to update other PR's.